operators/advanced_network/advanced_network/manager.cpp

```diff
@@ -112,9 +126,35 @@ ManagerType ManagerFactory::get_manager_type(const Config& config) {
 
 template ManagerType ManagerFactory::get_manager_type<Config>(const Config&);
 
+size_t Manager::get_alignment(MemoryKind kind) {
+  switch (kind) {
+    case MemoryKind::HOST:
+    case MemoryKind::HOST_PINNED:
+    case MemoryKind::HUGE:
+      return 128;
+    case MemoryKind::DEVICE:
+      return 256;
+    default:
+      return 128;
+  }
+}
+
+Status Manager::populate_pool(struct rte_ring* ring, const std::string& mr_name) {
```

Comment:
lots of pools, can you comment what this is for specifically? Is that what you want other backends to eventually reuse also?

Reply:
These are the pools for storing the data buffers in the MR. It is not what other backends would use since they could use something like a DPDK pktmbuf pool instead, whereas for RDMA we don't use mbufs and don't want the overhead/limitations that go along with it.

Reply:
`populate_generic_pool` ?

Doxygen on the role, what it creates, what it can be used for